analysis/Proposed_indicators.md

```diff
@@ -1,6 +1,83 @@
+## area in sqkm
+[x['area_sqkm'] for x in nh_estimates]
+
+## intersection density per sqkm
+[x['stats']['intersection_density_km'] for x in nh_estimates]
+
+## clean intersection density per sqkm
+[x['stats']['clean_intersection_density_km'] for x in nh_estimates]
+```
+So, that makes use of the osmnx stats functionality for calculating street connectivity.  
+
+The use of the function might be overkill; assuming we have a dataframe of sample points, if you coded it right there might even be a vectorised solution to carry out the operations, like extracting the 50m buffered unary union of 1600m network linestrings for a sample point as a geometry field and then use intersection analysis for each sample point's network buffer polygon with other pre-prepared resources:
+
+- use population raster (eg worldpop, or other) to derive a scaled-population density measure to serve as a proxy measure for dwelling density (e.g. take sum of population cells intersecting sausage buffer and divide by some indicative number for population per dwelling (e.g. 4) then standardise that result by dividing by the area of the sausage buffer in Hectares  --- so a result a-bit like dwellings per hectare (although technically, it isn't that.... i'm not saying this is valid, but it could be done and might serve for this purpose)
+
+- count clean network intersections generated by osmnx which intersect a sample point's sausage buffer and divide by area in km2 for street connectivity per square kilometre
+
+- if we pre-extract a series of destination POIs grouped into something like (pending discussion on how to translate this globally) 'supermarkets', 'convenience', 'public transport stops' (the latter from GTFS analysis - possibly restricted to those with 'frequent' service) then calculate a score for each sausage buffer based on presence of at least one from each category.
+
+Then, once you have a dataframe with rows per sample points with each associated with the above measures we could calculate their respective z-scores as per [this post](https://stackoverflow.com/questions/24761998/pandas-compute-z-score-for-all-columns) you could do something like
+
+```python
+indicators = ['dd_nh1600m','sc_nh1600m','dl_nh1600m']
+nh_zscores = list()
+for col in indicators
+    z = 'z_{}'.format(col)
+    df[z] = (df[col] - df[col].mean())/df[col].std(ddof=0)
+    nh_zscores.append(z)
+
+df['walkability'] = =df.loc[:,nh_zscores].sum(axis=1)
+```
+Then, summary statistics for small administrative areas, regular grid squares or hexagons can be calculated --- e.g. mean, sd, min, max, p25, p50, p75, as well as for the study region as a whole
+
+We then have average walkability for a given small area (be it an administrative area, or some standard area we use for all regions -- like hexagon for a hexbin plot) and can also see how it varies not only across areas but also within areas (if areas are reasonably, we would expect not much variation amongst the sample points).
```

Comment:
Does the small area you mention here mean the local neighbourhood derived through the sample points?